### PR TITLE
Update ca-certificates recipe

### PR DESCRIPTION
**Pull request by joaoflnedap**
Update `ca-certificates` recipe to the latest version, from `poky-zeus` (from October 2019)

### CODE REVIEW

File: meta/recipes-support/ca-certificates/ca-certificates_20190110.bb [hunk 1]
```diff
@@ -7,25 +7,23 @@ SECTION = "misc"
...
-           file://0001-update-ca-certificates-don-t-use-Debianisms-in-run-p.patch \
            file://0002-update-ca-certificates-use-SYSROOT.patch \
-           file://0003-update-ca-certificates-use-relative-symlinks-from-ET.patch \
-           file://0004-update-ca-certificates-use-c-rehash-instead-of-openssl-rehash.patch \
+           file://0001-update-ca-certificates-don-t-use-Debianisms-in-run-p.patch \
            file://update-ca-certificates-support-Toybox.patch \
            file://default-sysroot.patch \
            file://sbindir.patch \
+           file://0003-update-ca-certificates-use-relative-symlinks-from-ET.patch \
+           file://0001-certdata2pem.py-use-python3.patch \
```
Comment (bobhobelman) [inline meta/recipes-support/ca-certificates/ca-certificates_20190110.bb]: Seems not the convention to me if I see the numbers. And why are some without numbers?

File: meta/recipes-support/ca-certificates/ca-certificates_20190110.bb [hunk 2]
```diff
@@ -7,25 +7,23 @@ SECTION = "misc"
...
 LIC_FILES_CHKSUM = "file://debian/copyright;md5=aeb420429b1659507e0a5a1b123e8308"
 
-PR = "r1"
-
 # This is needed to ensure we can run the postinst at image creation time
 DEPENDS = ""
 DEPENDS_class-native = "openssl-native"
 DEPENDS_class-nativesdk = "openssl-native"
-# Need c_rehash from openssl and run-parts from debianutils
+# Need rehash from openssl and run-parts from debianutils
```
Comment (bobhobelman) [inline meta/recipes-support/ca-certificates/ca-certificates_20190110.bb]: Is there a specific reason to switch?

File: meta/recipes-support/ca-certificates/ca-certificates_20190110.bb [hunk 3]
```diff
@@ -7,25 +7,23 @@ SECTION = "misc"
 LICENSE = "GPL-2.0+ & MPL-2.0"
 LIC_FILES_CHKSUM = "file://debian/copyright;md5=aeb420429b1659507e0a5a1b123e8308"
 
-PR = "r1"
```
Comment (bobhobelman) [inline meta/recipes-support/ca-certificates/ca-certificates_20190110.bb]: I thought this was needed, or do we only needed if we change something?

File: meta/recipes-support/ca-certificates/ca-certificates_20190110.bb (hunk 1 above)
Comment (joaoflnedap) [reply meta/recipes-support/ca-certificates/ca-certificates_20190110.bb]: On my understanding, the numbering is used to ensure a given order of execution. I could omit the numbers from the recipe, while the files are still numbered, and still that wont affect its functionality.
In any case, I kept It as similar as possible to the original recipe (distributed on yocto-zeus).

**Pull request merged**